action/protocol/execution/evm/evm.go

```diff
@@ -45,6 +46,19 @@ func CanTransfer(db vm.StateDB, fromHash common.Address, balance *big.Int) bool
 func MakeTransfer(db vm.StateDB, fromHash, toHash common.Address, amount *big.Int) {
 	db.SubBalance(fromHash, amount)
 	db.AddBalance(toHash, amount)
+
+	outAddr, _ := address.FromBytes(fromHash.Bytes())
+	inAddr, _ := address.FromBytes(toHash.Bytes())
+	sdb, ok := db.(*StateDBAdapter)
+	if !ok {
+		log.L().Panic("failed to get StateDBAdapter")
+	}
+	systemLog := systemlog.LogEvmTransfer(&systemlog.EvmTransfer{
+		Amount: amount,
+		From:   outAddr.String(),
+		To:     inAddr.String(),
+	})
+	sdb.AddSystemLog(systemLog)
```

Comment:
Define a global topic
InContractTransfer := make([]byte, 8, 8) // 8 bytes with all 0s
```
db.AddLog(&types.Log{
  Topics: [InContractTransfer, fromHash, toHash],
   Data: []byte{amount},
})
```

Reply:
Ok. I was not sure whether there would be a conflict with native topic, so I didn't use zero bytes topic before.